graphql_compiler/compiler/emit_sql.py

```diff
@@ -205,25 +289,66 @@ def _set_group_by(self, group_by):
         """Set output columns for the fold object."""
         self._group_by = group_by
 
-    def _construct_fold_joins(self, edge, from_alias, to_alias):
+    def _construct_fold_joins(self):
```

Comment:
Are we supposed to be mutating `self.join_info` here? Can we guarantee that this is safe to do? Do we gain anything by mutating it?